kafka/src/main/scala/hydra/kafka/services/StreamsManagerActor.scala

```diff
@@ -50,22 +50,22 @@ class StreamsManagerActor(bootstrapKafkaConfig: Config,
   override def receive: Receive = Actor.emptyBehavior
 
   override def preStart(): Unit = {
-    context.become(streaming(metadataStream.run()))
+    context.become(streaming(metadataStream.run(), Map.empty))
   }
 
 
-  def streaming(stream: (Control, NotUsed)): Receive = {
+  def streaming(stream: (Control, NotUsed), metadataMap: Map[String, TopicMetadata]): Receive = {
```

Comment:
we now pass a new map forward instead of using a private member var, to follow a more immutable style